static/js/components/Toast.js

```diff
@@ -30,7 +30,7 @@ export default class Toast extends React.Component {
   render() {
     const { children, open } = this.props;
 
-    return <div className={`toast ${open ? 'open' : ''}`}>
+    return <div role="alert" className={`toast ${open ? 'open' : ''}`}>
```

Comment:
That sounds reasonable to me, we could move the toast rendering function into a helper method and have the `render` function just do something like

```js
return open ? this.toast() : null;
```

which would take care of it.

Wondering what @noisecapella thinks about this.